avocado/utils/build.py

```diff
@@ -10,17 +10,18 @@
 # See LICENSE for more details.
 #
 # Copyright: Red Hat Inc. 2013-2014
+#          : IBM 2016
 # Author: Lucas Meneghel Rodrigues <lmr@redhat.com>
+#       : Praveen K Pandey <praveen@linux.vnet.ibm.com>
```

Comment:
Well I wouldn't call this a big change, would you? (the purpose of authorship is to mark significant changes like new API, complete reworks etc. This change is only a minor modification which does not require the change of copyright)